src/dmd/backend/dwarf.d

```diff
@@ -18,6 +18,7 @@ import dmd.backend.type;
 extern (C++):
 
 nothrow:
+@safe:
```

Comment:
Same here, those are trusted

Reply:
same answer, buildkite dstyle rejects `@trusted:` and `@trusted { ... }`